airbyte-integrations/connectors/source-shopify/source_shopify/streams/streams.py

```diff
@@ -167,22 +168,16 @@ class MetafieldProducts(IncrementalShopifyGraphQlBulkStream):
     bulk_query: MetafieldProduct = MetafieldProduct
 
 
-class ProductImages(IncrementalShopifyNestedStream):
-    parent_stream_class = Products
-    nested_entity = "images"
-    # add `product_id` to each nested subrecord
-    mutation_map = {"product_id": "id"}
+class ProductImages(IncrementalShopifyGraphQlBulkStream):
+    bulk_query: ProductImage = ProductImage
 
 
 class MetafieldProductImages(IncrementalShopifyGraphQlBulkStream):
     bulk_query: MetafieldProductImage = MetafieldProductImage
 
 
-class ProductVariants(IncrementalShopifyNestedStream):
-    parent_stream_class = Products
-    nested_entity = "variants"
-    # add `product_id` to each nested subrecord
-    mutation_map = {"product_id": "id"}
+class ProductVariants(IncrementalShopifyGraphQlBulkStream):
```

Comment:
There are a couple of change with that one. For example, `inventory_policy` goes from `deny` to `DENY`, `inventory_management` goes from `shopify` to `SHOPIFY`, etc... should we fix those or else be explicit in the migration guide?

Reply:
What difference does it create? We had lower-case string values - now they are upper-case, but the schema for these fields hasn't changed. I don't see it as any kind of `breaking` in any scenario. GraphQL just returns some values like this.

Reply:
Added to the migration-guide